### PR TITLE
adding update for publishing the CVE patched kernel

### PR DESCRIPTION
**Pull request by bennyvasquez**

### CODE REVIEW

File: content/blog/2024-04-02-xz-and-cve-2024-1086.md [hunk 1]
```diff
@@ -14,6 +14,29 @@ post:
...
 
+*Update, April 3rd, 2024*
+
+The patches for CVE-2024-1086 are now available in production repos. 
+
+For AlmaLinux 8, you are looking for kernel-4.18.0-513.18.2.el8_9 higher.
+
+```bash
+[root@host ~]# rpm -qa kernel
+kernel-4.18.0-513.18.2.el8_9_3.x86_64
```
Comment (andrewlukoshko) [inline content/blog/2024-04-02-xz-and-cve-2024-1086.md:25]: el8_9_3 -> el8_9

**Review by andrewlukoshko — CHANGES_REQUESTED** [review]

File: content/blog/2024-04-02-xz-and-cve-2024-1086.md (hunk 1 above)
Comment (jonathanspw) [reply content/blog/2024-04-02-xz-and-cve-2024-1086.md:25]: `for kernel-4.18.0-513.18.2.el8_9 higher`

Missing an "or"

**Comment by jonathanspw** [comment]
lgtm

**Review by jonathanspw — APPROVED** [review]
lgtm

**Pull request merged**